lib/output/utils.js

```diff
@@ -42,13 +39,13 @@ function initCtorRegistry(globalObject) {
   }
 
   const ctorRegistry = Object.create(null);
-  for (const intrinsic of intrinsicConstructors) {
-    ctorRegistry[`%${intrinsic}%`] = globalObject[intrinsic];
```

Comment:
Refs <https://github.com/jsdom/webidl2js/pull/242#discussion_r606838290>:
```suggestion
    ctorRegistry[intrinsic] = globalObject[intrinsic];
```

Reply:
You are commenting on a deleted line.